sdcflows/interfaces/bspline.py

```diff
@@ -621,5 +635,10 @@ def _fix_topup_fieldcoeff(in_coeff, fmap_ref, out_file=None):
     coeffnii.header.set_qform(coeffnii.header.get_qform(coded=False), code=0)
     coeffnii.header.set_sform(newaff, code=1)
 
+    pe_axis = "ijk".index(pe_dir[0])
+    pe_sign = [1.0, -1.0][len(pe_dir) != 2]
```

Comment:
is this flipped? in any case, I think this can be a little more explicit
```suggestion
    pe_sign = -1.0 if pe_dir.endswith('-') else 1.0
```

Reply:
Just to confirm - my suggestion changed the behavior (i.e., `j- == -1.0`)

Reply:
Yes, I believe you are right.